client/presentation/BUILD

```diff
@@ -21,24 +21,23 @@ rust_library(
     srcs = ["sdl2_vulkan_interop.rs"],
     deps = [
         ":vulkan",
-        "//third_party/SDL2:sdl2_sys",
         "//cargo:sdl2",
         "//cargo:vk_sys",
+        "//cargo/overrides/sdl2-sys:sdl2_sys",
```

Comment:
what happened to third party?

Reply:
I moved the bindgen invocation and rust_library out of third_party (where it kind-of didn't belong), and put it in cargo/overrides where people would expect it to be.

As a recap, this is a me-generated override for sdl2-sys.